source/examples/rapids-morpheus-pipeline/notebook.md

```diff
@@ -0,0 +1,164 @@
+and publishes network data to the Kafka topic. This producer script is containerized using a custom Docker image that can be built using the Dockerfile in the `scripts/producer-dockerfile` directory.
+
+- Uses `kafka-python` for message production.
+- Contains the producer script for generating network data.
+
+### Triton-Morpheus Deployment (`k8s/triton`)
+
+The inference server is deployed using the NVIDIA Morpheus- Triton Inference Server docker image
+`nvcr.io/nvidia/morpheus/morpheus-tritonserver-models:25.02`.
+
+### Morpheus Pipeline Deployment (`k8s/morpheus-pipeline`)
+
+The core processing pipeline is deployed as a separate pod that, uses an image that can be built using the Dockerfile in
+the `pipeline-dockerfile` directory.
+
+- Runs the Morpheus nightly 25.06 conda build
+- Contains all pipeline and stage scripts `scripts/pipeline-dockerfile/*.py`
+- Processes the streaming data through the various stages
+
+(deploying-on-eks)=
+
+## Deploying on EKS
+
+### Prerequisites
+
+You need to have the [`aws` CLI tool](https://aws.amazon.com/cli/) and [`eksctl` CLI tool](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) installed along with [`kubectl`](https://kubernetes.io/docs/tasks/tools/) for managing Kubernetes.
+
+### Launch GPU enabled EKS cluster
+
+We launch a GPU enabled EKS cluster with `eksctl`.
+
+```{note}
+1. You will need to create or import a public SSH key to be able to execute the following command.
+In your aws console under `EC2` in the side panel under Network & Security > Key Pairs, you can create a
+key pair or import (see "Actions" dropdown) one you've created locally.
+
+2. If you are not using your default AWS profile, add `--profile <your-profile>` to the following command.
+```
+
+```console
+$ eksctl create cluster morpheus-rapids \
+    --version 1.32 \
+    --nodes 3 \
+    --node-type=g4dn.xlarge \
+    --timeout=40m \
+    --ssh-access \
+    --ssh-public-key  <public key ID> \  # Name assigned during creation of your key in aws console\
+    --region us-east-1 \
+    --zones=us-east-1c,us-east-1b,us-east-1d \
+    --auto-kubeconfig
+```
+
+To access the cluster we need to pull down the credentials. Add `--profile <your-profile>` if you are not using the
+default profile.
+
+```console
+$ aws eks --region us-east-1 update-kubeconfig --name morpheus-rapids
+```
+
+### Deploy the Strimzi Operator
+
+Strimzi is an open-source project that provides a way to run Apache Kafka on Kubernetes. It
+simplifies the deployment and management of Kafka clusters by providing Kubernetes operators that
+handle the complex tasks of setting up and maintaining Kafka.
+
+We use `kubectl` to deploy teh operator. In our case we are deploying everything on the default
+namespace, and the entire pipeline is design for that.
+
+```console
+$ kubectl create -f 'https://strimzi.io/install/latest?namespace=default'
```

Comment:
Specifying the namespace this way is a little unusual. Would this more idiomatic kubernetes method not work?

```suggestion
$ kubectl create -n default -f 'https://strimzi.io/install/latest'
```

Reply:
I was following their docs. I noticed when I tried running 
```
$ kubectl create  -f 'https://strimzi.io/install/latest'
```
which would be the default, it didn't deployed properly, and then the cluster later one was never up.  But I will double check. I might have have a typo

Reply:
If it's necessary then that's totally fine, just wanted to check we were doing is an unusual way for a good reason.